build.gradle

```diff
@@ -70,7 +70,7 @@ allprojects {
 
     test {
         reports.html.enabled = false
-        reports.junitXml.destination = file(Paths.get(rootProject.buildDir.toString(), "tests", project.name))
+        ignoreFailures true
```

Comment:
I think you don't want to do this, then if we run test locally, it would not fail

I think we should use `-Dtest.ignoreFailures=true` just for travis